CrisRorens07/clbs
# Feature/cris rorens07/fe 890 detail page

# Related Tickets
FE-890

# Description

- Add chateristdesc.html, coviddesc.html, kbdmusic.html, kobetixdesc.html
- Update index.html, about.html and style.css
- Add and Change some icons and pictures

# Verification

1. check the code, there should be a:
- chateristdesc.html
-  coviddesc.html
- kbdmusic.html
- kobetixdesc.html

2. The index.html, about.html and style.css should be updated
3. check if there are new icons and pictures

# Risks
some icons/links are not yet arranged and is still messed up
